### PR TITLE
Adds a Waste and Distribution Monitor to DeltaStation

### PR DESCRIPTION
**Pull request by haydenredacted**
## What Does This PR Do

Deltastation for whatever reason does not have a computer to monitor the distribution pressure and air mix tanks. This PR adds that while also fixing various air sensors and meters to have the proper variable so they get detected by the monitor.

## Why It's Good For The Game

Monitoring atmos is good and you'll know if the crew will yell at you when there's vent crawling antagonists because the vents are bursting.

## Images of changes

In game:
image

StrongDMM:
image

## Testing

Loaded Deltastation and looked at the pressure of the distro
## Changelog
:cl:
add: Added Waste and Distribution monitor to atmospherics on Deltastation
tweak: Changed variables of air meters/sensors to be detectable by the monitor
/:cl:

### CODE REVIEW

**Comment by haydenredacted** [comment]
Please excuse my constant brawling with this config file, my other PR had the same issue.

**Comment by HMBGERDO** [comment]
you have `config/config.toml` added in PR files, you should remove it

File: config/config.toml
Comment (Sirryan2002) [inline config/config.toml:1]: this needs to be removed from your changes :)

**Review by Sirryan2002 — CHANGES_REQUESTED** [review]

**Comment by haydenredacted** [comment]
config.toml removed.

**Review by AffectedArc07 — APPROVED** [review]

**Review by Sirryan2002 — APPROVED** [review]

**Review by lewcc — APPROVED** [review]

**Pull request merged**